app-text/pdf2djvu/pdf2djvu-0.9.17.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit python-any-r1 toolchain-funcs flag-o-matic
+
+DESCRIPTION="A tool to create DjVu files from PDF files"
+HOMEPAGE="http://jwilk.net/software/pdf2djvu"
+SRC_URI="https://github.com/jwilk/${PN}/releases/download/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+graphicsmagick nls openmp test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=app-text/djvu-3.5.21:=
+	>=app-text/poppler-0.16.7:=
+	dev-libs/libxml2:=
+	dev-libs/libxslt:=
+	graphicsmagick? ( media-gfx/graphicsmagick:= )
```

Comment:
Are you aware how this is used in the build system? Are the libs/headers used, or is it executed? May be a BDEPEND too.